.github/workflows/update-canary.yml

```diff
@@ -0,0 +1,34 @@
+name: Update canary branch
+
+on:
+  schedule:
+    - cron: '0 6 * * *'
```

Comment:
Also, other then schedule, I hope there is a way to manually trigger this like webook events.

Reply:
https://docs.github.com/en/actions/managing-workflow-runs/manually-running-a-workflow

I should really test this some time, it looks cool

Reply:
I wouldn't block this PR on it though, incremental improvements let us experiment faster :)

Reply:
> I wouldn't block this PR on it though, incremental improvements let us experiment faster :)

Yeap. this shouldn't be a blocker. But we can keep jenkins job before it's done (remove schedule part in jenkins job).

Reply:
I added the workflow_dispatch event